code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -803,6 +803,18 @@
 	. = 1
 	..()
 
+/datum/reagent/toxin/acid/nitracid
+	name = "Nitric acid"
+	description = "Nitric acid is an extremely corrosive chemical substance that violently reacts with living organic tissue."
+	color = "#5050FF"
+	toxpwr = 6
+	acidpwr = 5.0
+
+/datum/reagent/toxin/acid/nitracid/on_mob_life(mob/living/carbon/M)
+	M.adjustFireLoss(current_cycle/12, 0)
```

Comment:
Second argument is a bool.

Reply:
> What exactly is the point of adding multiple new chems that either don't do anything or do basically the same thing?
> Why exactly do we need more murder chems?
> Please atomize this.

variety mainly. Also because i wanted to nerf these extremely easy to create explosives. These are more powerful but also very finnicky to deal with.